### PR TITLE
Re-adds and adds a few more items to loadout

### PR DESCRIPTION
**Pull request by chesse20**
## About The Pull Request

Adds a bunch of items and some new items from before the loadout refactor. Includes plushies, lipstick, skateboard, instrument beacon, and breath masks plus a security mask

## Why It's Good For The Game

I can have a Purple vox breathing mask to match my purple skirt and also everyone can have some funny plushies to play with.

## Changelog

:cl:
add: A bunch of old loadout items
add: A few new masks 
fix: You can see which jobs let you use certain items

/:cl:

### CODE REVIEW

File: code/modules/client/preferences/loadout/loadout_item/backpack.dm [hunk 1]
```diff
@@ -38,6 +38,56 @@
...
+	subcategory = LOADOUT_SUBCATEGORY_BACKPACK_TOYS
+
+/datum/loadout_item/backpack/snake_plushie
+	path = /obj/item/toy/plush/snakeplushie
+	subcategory = LOADOUT_SUBCATEGORY_BACKPACK_TOYS
+
+/datum/loadout_item/backpack/skateboard
+	path = /obj/item/melee/skateboard
+	cost = 5 //lets you ignore hunger and the weight of spacesuits and its a weapon
+	customization_flags = CUSTOMIZE_NONE //all customization is lost when you put the skateboard on the ground because of how skateboards are implemented
```
Comment (Kapu1178) [inline code/modules/client/preferences/loadout/loadout_item/backpack.dm:85]: I don't think I want this to be accessible roundstart

File: code/modules/client/preferences/loadout/loadout_item/mask.dm [hunk 2]
```diff
@@ -34,3 +34,14 @@
...
+/datum/loadout_item/mask/breath_mask
+	path = /obj/item/clothing/mask/breath
+
+/datum/loadout_item/mask/breath_mask_vox
+	path = /obj/item/clothing/mask/breath/vox
+
+/datum/loadout_item/mask/security_hailer
+	path = /obj/item/clothing/mask/gas/sechailer
+	restricted_roles = list(JOB_SECURITY_OFFICER,JOB_HEAD_OF_SECURITY,JOB_WARDEN)
+	customization_flags = CUSTOMIZE_NAME_DESC //has no parts to recolor
```
Comment (Kapu1178) [inline code/modules/client/preferences/loadout/loadout_item/mask.dm:46]: Sec gets these in their vendor, why do they need a loadout item for it?

File: code/__DEFINES/loadout.dm [hunk 3]
```diff
@@ -59,6 +59,7 @@
 #define MAX_LOADOUT_SLOTS 6
 
 // Flags for customizing the loadout items
+#define CUSTOMIZE_NONE 0
```
Comment (Kapu1178) [inline code/__DEFINES/loadout.dm:62]: use NONE, this flag isnt needed

**Review by Kapu1178 — CHANGES_REQUESTED** [review]

File: code/modules/client/preferences/loadout/loadout_item/mask.dm (hunk 2 above)
Comment (chesse20) [reply code/modules/client/preferences/loadout/loadout_item/mask.dm:46]: i forgot it didn't have recoloring while putting it in load out, i think i confused it with the swat version

Comment (Kapu1178) [reply code/modules/client/preferences/loadout/loadout_item/mask.dm]: Any item can be color rotated, for the record. Up to you if you want it.

Comment (chesse20) [reply code/modules/client/preferences/loadout/loadout_item/mask.dm]: i mean i tested the security hailer with color rotation it didn't do anything so no real point to have it in loadout

Comment (Kapu1178) [reply code/modules/client/preferences/loadout/loadout_item/mask.dm]: Oh its probably because its black.

**Review by Kapu1178 — APPROVED** [review]

**Pull request merged**